examples/benchmark.py

```diff
@@ -16,7 +16,7 @@ def reset(self):
 
     def act(self, observations):
         action = SimulatorActions.MOVE_FORWARD
-        return action
+        return action, None
```

Comment:
Suggestion: Can we move to dict based input and output system now? Adding more arguments and return items to the signatures is cumbersome and requires doing a lot of changes to make sure there is no regression. Moving to dict based output and input system will remove these concerns. I foresee in future that if we need to add more arguments and return items, we will have to do this again. 

So, I suggest moving to dict based output, input structure. I moved to it in Pythia and life has been a breeze since then.